### PR TITLE
Fix uploader module

### PR DESCRIPTION
**Pull request by m4ksio**
Turns out we cannot use single variable.
Fix propagating the module

### CODE REVIEW

**Review by vishalchangrani — APPROVED** [review]

**Review by huitseeker — APPROVED** [review]

**Review by huitseeker — CHANGES_REQUESTED** [review]
You may want to fix the comment.

File: engine/execution/computation/computer/uploader/uploader_test.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ func Test_AsyncUploader(t *testing.T) {
 
 func Test_GCPBucketUploader(t *testing.T) {
 
-	t.Skip("requires GCP Bucket setup")
+	//t.Skip("requires GCP Bucket setup")
```
Comment (huitseeker) [inline engine/execution/computation/computer/uploader/uploader_test.go:126]: Are you sure?

Comment (m4ksio) [reply engine/execution/computation/computer/uploader/uploader_test.go:126]: Of course not.
We need CI for branches badly.
Thanks for catching this

**Review by huitseeker — APPROVED** [review]
Thanks!

**Pull request merged**